lms/services/lti_grading/_v13.py

```diff
@@ -54,5 +55,20 @@ def record_result(self, grading_id, score=None):  # pylint:disable=arguments-dif
                 "activityProgress": "Completed",
                 "gradingProgress": "FullyGraded",
             },
-            headers={"Content-Type": "application/vnd.ims.lis.v2.lineitem+json"},
+            headers={"Content-Type": "application/vnd.ims.lis.v1.score+json"},
```

Comment:
This values was wrong: https://www.imsglobal.org/spec/lti-ags/v2p0#example-posting-a-final-score-update

I guess both blackboard and IMS didn't care about that. Moodle does. I retested in the other ones and still works with the right value.